### ISSUE TITLE
Add a new ZbState called ZbLoaded to indicate that all ZbDevices and ZbLoads have been loaded successfully.

### ISSUE THREAD

**Issue by tasict** [arendst/Tasmota#21305]
I hope to add a new state to indicate that all ZbDevices and ZbLoads have been loaded successfully.

**Comment by sfromis** [comment]
I's say that it should be safe to use the existing `{"ZbState":{"Status":0,"Message":"Started"}}` with an extra one-second interval for the plugin data to be loaded from the local file system.

**Comment by tasict** [comment]
No, at this point, the ZB device and zb driver have not actually finished loading yet. If ZbState#Status=0 is added in the rule, calling Zbinfo at this time will fail to display information properly. It requires a delay of about 2 to 3 seconds before calling zbinfo. This is exactly why there is a need to add a new state in reality.

**Comment by s-hadinger** [comment]
Got it. I will add it

**Comment by s-hadinger** [comment]
Or move Started event after the plug-ins are loaded, which sounds better

**Comment by tasict** [comment]
Indeed, that would be perfect!
